integration_tests/e2e/refer.cy.ts

```diff
@@ -83,13 +79,6 @@ context('Refer', () => {
       setting: 'Custody',
     })
```

Comment:
I can't remember where we got to in our thinking around using utils in tests (was that about unit tests?) but here we could just say `const person = PersonUtils.personFromPrisoner(prisoner)` rather than recreating it?

It was like this before, so maybe we concluded this was better?

Reply:
Yeah I think I wanted to test the whole integration here without relying on our internal implementation, hence creating the expected Person

Reply:
Makes sense - I guess this strengthens the case for avoiding POS language above?

Reply:
Yep, have switched that above